src/features/profile/Profile.tsx

```diff
@@ -0,0 +1,57 @@
+import React, { useState } from 'react'
+import ProfileGeneral from './myProfile/MyProfile'
+import MySecurity from './mySecurity/MySecurity'
+import PageHeader from '../../common/components/PageHeader'
+import Breadcrumbs, { TBreadcrumb } from '../../common/components/Breadcrumbs'
+
+type TProfileProps = {
+  info: {
+    currencyName: string
+  }
+}
+
+const Profile = (props: TProfileProps): JSX.Element => {
+  const { info } = props
+  const [selectIndex, setSelectIndex] = useState(0)
+
+  const tabs = [
+    { label: 'Info', count: 2, component: <ProfileGeneral /> },
+    {
+      label: 'Comments',
+      count: 26,
+      component: <div className='text-center'> No comments </div>,
+    },
+    { label: 'Security', component: <MySecurity info={info} /> },
+  ]
```

Comment:
In such way all tabs will be rendered, please change it to conditional rendering like

```
{selectIndex === 0 && <ProfileGeneral />}
```

Could you also add a typescript enum instead of number to make it easier to understand:
```
enum Tab {
  info,
  comments,
  security,
}

const [tab, setTab] = useState(Tab.info)

...
{tab === Tab.info && <ProfileGeneral />}
```